### DISCUSSION TITLE
Error! When I run multiple job

### DISCUSSION THREAD

**Discussion by ThietBHL** [swar/Swar-Chia-Plot-Manager#879] [Q&A]
![image](https://user-images.githubusercontent.com/84167544/119526741-d96f3a00-bda9-11eb-8320-4ec668dcd1e8.png)

My config as below. Please help

  - name: chia-01
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: D:\Temp1
    temporary2_directory:
    destination_directory: E:\
    size: 32
    bitfield: true
    threads: 3
    buckets: 128
    memory_buffer: 4000
    max_concurrent: 6
    max_concurrent_with_start_early: 7
    initial_delay_minutes: 0
    stagger_minutes: 60
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23 ]

- name: chia-02
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: D:\Temp2
    temporary2_directory:
    destination_directory: E:\
    size: 32
    bitfield: true
    threads: 3
    buckets: 128
    memory_buffer: 4000
    max_concurrent: 6
    max_concurrent_with_start_early: 7
    initial_delay_minutes: 0
    stagger_minutes: 60
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23 ]

**Comment by Farmer-Chia** [comment]
Hi! Have you solved the problem? I have the same

**Comment by Chilli71** [comment]
Please. in future use the 'insert code' function if you want to paste your configuration.
The way you pasted it, it would be next to impossible to find format failures.
The used parser has some limitations eg. you must not use 'tab' for the text alignment.

**Comment by VikingoGarage** [comment]
Hi! Have you solved the problem? I have the same

**Comment by SjA-nl** [comment]
destination_directory: E: 
probably must be as  destination_directory: E:\

**Reply by Chilli71** [reply]
I have another problem on another machine and I didn't find that it makes a difference if there is a backslash at the end of the directory or not.
If so all other directories would need to be updated with backlash, too.

**Reply by SjA-nl** [reply]
> I have another problem on another machine and I didn't find that it makes a difference if there is a backslash at the end of the directory or not.
> If so all other directories would need to be updated with backlash, too.

" backlash" need, I twice checked now.
